src/elements/content-sidebar/BoxAISidebar.scss

```diff
@@ -0,0 +1,11 @@
+@import '~@box/blueprint-web-assets/tokens/tokens.scss';
+
+.bcs-BoxAISidebar {
+    .bcs-content-header {
+        justify-content: left;
```

Comment:
Are we certain we wanna push it to the left instead of the right?, the other tabs have buttons coming from right to left instead of left to right

Reply:
There will be more controls added to this header, that's why it's aligned to left right now.

Reply:
Even if we don't have all the controls currently, can we still have the proper formatting in place so we don't miss it? We are looking to have the expand button on the right side right? So something like a `space-between`?